VSDiagnostics/VSDiagnostics/VSDiagnostics.Test/Tests/General/FieldCanBeReadonlyTests.cs

```diff
@@ -0,0 +1,361 @@
+﻿using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using RoslynTester.Helpers.CSharp;
+using VSDiagnostics.Diagnostics.General.FieldCanBeReadonly;
+
+namespace VSDiagnostics.Test.Tests.General
+{
+    [TestClass]
+    public class FieldCanBeReadonlyTests : CSharpCodeFixVerifier
+    {
+        protected override DiagnosticAnalyzer DiagnosticAnalyzer => new FieldCanBeReadonlyAnalyzer();
+
+        protected override CodeFixProvider CodeFixProvider => new FieldCanBeReadonlyCodeFix();
+
+        [TestMethod]
+        public void FieldCanBeReadonly_AssignedInline()
+        {
+            var original = @"
+namespace ConsoleApplication1
+{
+    class MyClass
+    {
+        private int _foo = 0;
+    }
+}";
+
+            var result = @"
+namespace ConsoleApplication1
+{
+    class MyClass
+    {
+        private readonly int _foo = 0;
+    }
+}";
+
+            VerifyDiagnostic(original, string.Format(FieldCanBeReadonlyAnalyzer.Rule.MessageFormat.ToString(), "_foo"));
+            VerifyFix(original, result);
+        }
+
+        [TestMethod]
+        public void MultipleFieldsCanBeReadonly_AssignedInline()
+        {
+            var original = @"
+namespace ConsoleApplication1
+{
+    class MyClass
+    {
+        private int _foo = 0, _bar = 1;
+    }
+}";
+            var result = @"
+namespace ConsoleApplication1
+{
+    class MyClass
+    {
+        private readonly int _foo = 0, _bar = 1;
+    }
+}";
+
+            VerifyDiagnostic(original,
+                string.Format(FieldCanBeReadonlyAnalyzer.Rule.MessageFormat.ToString(), "_foo"),
+                string.Format(FieldCanBeReadonlyAnalyzer.Rule.MessageFormat.ToString(), "_bar"));
+            VerifyFix(original, result);
```

Comment:
Why not `VerifyFix(original, original);`? Clearer & shorter IMO.
Or maybe a `VerifyNotFixed(original);`?